client/ReviewModule.jsx

```diff
@@ -1,36 +1,26 @@
 import React from 'react';
+import $ from 'jquery';
+import axios from 'axios';
 import TopBar from './TopBar.jsx';
 import Reviews from './Reviews.jsx';
 
 export default class ReviewModule extends React.Component {
   constructor() {
     super();
-
     this.state = {
       searchText: '',
       reviews: [
         {
-          locName: "the barbed wire",
+          locname: 'Intial',
           stars: 3,
           posted: '2018-12-25',
           username: 'Tim, the user',
-          userLoc: 'San Francisco, CA',
+          userloc: 'San Francisco, CA',
           numfriends: 15,
           photoLoc: '../photos/p1.jpg',
-          mumReviews: 5,
+          mumreviews: 5,
```

Comment:
Is this supposed to be 'mumreviews'? Not sure what 'mum' refers to, or why it is not camelCase. Just wanted to make sure you're certain!